apps/ide/src/plugins/project-wizard/build/build.js

```diff
@@ -14,11 +14,20 @@
  * limitations under the License.
  */
 
+/**
+ * @file Build runner
+ * @since 1.0.0
+ * @author cimfalab@gmail.com
```

Comment:
Followed by our git [convention](http://confluence.score/display/ODP/Git+Convention) and last talking with wcho, using public email address(ooo@samsung.com) is recommended.
Why don't you change your GitHub email address for this time? :smile: 

> 1) 커밋시 author 이름 설정(회사에서 공식적으로 이용하는 영문 이름으로) 및 Github 프로파일 이름과 맞추기 누가 누군지 알기 어렵고 장기적으로 외부와의 협업도 고려하면 formal한 형태의 이름 하나를 쓰는 게 좋습니다. - wcho -